config/default/kustomization.yaml

```diff
@@ -0,0 +1,144 @@
+# Adds namespace to all resources.
+namespace: lightspeed-operator
```

Comment:
i think we will want this to be `openshift-lightspeed-operator`, to distinguish from ansible lightspeed.  Plus being in an openshift prefixed namespace allows our metrics to be reported to telemetry